modules/cluster-cloud-controller-manager-operator.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+[id="cluster-cloud-controller-manager-operator_{context}"]
+= Cluster Cloud Controller Manager Operator
+
+[discrete]
+== Purpose
+
+The Cluster Cloud Controller Manager Operator manages and updates the cloud controller managers deployed on top of {product-title}. The Operator is based on the Kubebuilder framework and `controller-runtime` libraries. It is installed via the Cluster Version Operator (CVO).
+
+It contains the following components:
+
+* Operator
+* Bootstrap manifest renderer
+* Installer based on static pods
+* Configuration observer
```

Comment:
i'm not sure if we consider the config sync controller part of this, perhaps @lobziik could add a little more info

Reply:
technically, it is a separate controller.

Maybe change this line a bit for being more specific? like `Cloud configuration observer`?

Our dev doc for bit context: https://github.com/openshift/cluster-cloud-controller-manager-operator/blob/master/docs/dev/cloud-config-sync.md